legal-api/tests/unit/resources/v2/test_filing_comments.py

```diff
@@ -160,6 +160,7 @@ def test_get_comments_invalid_business_error(session, client, jwt):
         assert "CP2222222 not found" == rv.json.get("message")
 
 
+# TODO: Fix this
 def test_get_comments_invalid_filing_error(session, client, jwt):
```

Comment:
what exactly needs to be fixed here?

Reply:
Fixed it now. thanks for catching. Added a check to make sure `filing` is found in `filing_comments.py`